DuckDuckGo/MainViewController+KeyCommands.swift

```diff
@@ -70,7 +70,15 @@ extension MainViewController {
             ]
         }
 
-        return alwaysAvailable + browsingCommands + findInPageCommands + [
+        var arrowKeys = [UIKeyCommand]()
+        if omniBar.textField.isFirstResponder {
```

Comment:
If the arrow keys are always associated with a command it stops text editing in the web view.